checks.xml

```diff
@@ -166,10 +166,14 @@
     <!-- See https://checkstyle.org/config_coding.html -->
     <module name="EmptyStatement"/>
     <module name="EqualsHashCode"/>
-    <module name="HiddenField"/>
+    <module name="HiddenField">
+      <property name="tokens" value="VARIABLE_DEF"/>
```

Comment:
Why are we adding this exception?

Reply:
When we add constructors, the hiddenfield error shows up (since the parameter is the same as the field). If we look at the DrivePower class as an example, when we set the leftPower, the method parameter is leftPower and the method just does this.leftPower = leftPower. This prompts a HiddenField error, but I think it should be allowed in this instance.